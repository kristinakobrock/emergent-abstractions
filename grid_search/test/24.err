  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 26.54it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 24.62it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 30.64it/s] 28%|#########################7                                                                  | 14/50 [00:00<00:01, 31.84it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 29.32it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 27.15it/s] 48%|############################################1                                               | 24/50 [00:00<00:00, 26.70it/s] 54%|#################################################6                                          | 27/50 [00:00<00:00, 26.45it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 26.20it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 25.83it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 25.17it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 26.37it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 23.95it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 24.43it/s] 96%|########################################################################################3   | 48/50 [00:01<00:00, 25.31it/s]100%|############################################################################################| 50/50 [00:01<00:00, 26.24it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 31%|############################6                                                                | 4/13 [00:00<00:00, 26.48it/s] 54%|##################################################                                           | 7/13 [00:00<00:00, 24.19it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 23.25it/s]100%|############################################################################################| 13/13 [00:00<00:00, 21.90it/s]100%|############################################################################################| 13/13 [00:00<00:00, 22.77it/s]
