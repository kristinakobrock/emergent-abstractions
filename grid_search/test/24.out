# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6801122426986694, "acc": 0.6048873066902161, "length": 3.4884510040283203, "mode": "train", "epoch": 1}
{"loss": 0.6820871829986572, "acc": 0.5630859732627869, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6580023169517517, "acc": 0.6326470971107483, "length": 3.5111844539642334, "mode": "train", "epoch": 2}
{"loss": 0.6648036241531372, "acc": 0.6123230457305908, "length": 1.3984375, "mode": "test", "epoch": 2}
{"loss": 0.6559991836547852, "acc": 0.6329935789108276, "length": 3.537249803543091, "mode": "train", "epoch": 3}
{"loss": 0.6586405038833618, "acc": 0.625, "length": 4.0, "mode": "test", "epoch": 3}
{"loss": 0.6540880799293518, "acc": 0.6338396668434143, "length": 3.6131608486175537, "mode": "train", "epoch": 4}
{"loss": 0.6521574258804321, "acc": 0.630664050579071, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.6464626789093018, "acc": 0.6407861709594727, "length": 3.773970127105713, "mode": "train", "epoch": 5}
{"loss": 0.6349373459815979, "acc": 0.6509765982627869, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.6150857210159302, "acc": 0.6649749875068665, "length": 3.8859360218048096, "mode": "train", "epoch": 6}
{"loss": 0.5949839353561401, "acc": 0.6875, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.5786451697349548, "acc": 0.7020470499992371, "length": 3.9564576148986816, "mode": "train", "epoch": 7}
{"loss": 0.5846262574195862, "acc": 0.6998047232627869, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.5575799942016602, "acc": 0.7102963924407959, "length": 3.9601168632507324, "mode": "train", "epoch": 8}
{"loss": 0.5820807218551636, "acc": 0.7082031965255737, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.544782280921936, "acc": 0.722648561000824, "length": 3.969080686569214, "mode": "train", "epoch": 9}
{"loss": 0.5679219365119934, "acc": 0.7138672471046448, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.5360366702079773, "acc": 0.7287247776985168, "length": 3.9784748554229736, "mode": "train", "epoch": 10}
{"loss": 0.5567053556442261, "acc": 0.7226563096046448, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.515448272228241, "acc": 0.7421641945838928, "length": 3.9811084270477295, "mode": "train", "epoch": 11}
{"loss": 0.5349313616752625, "acc": 0.7429687976837158, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.49818843603134155, "acc": 0.7522565722465515, "length": 3.989014148712158, "mode": "train", "epoch": 12}
{"loss": 0.5227241516113281, "acc": 0.7509765625, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.48361459374427795, "acc": 0.7621640563011169, "length": 3.9930810928344727, "mode": "train", "epoch": 13}
{"loss": 0.5216273069381714, "acc": 0.7482421398162842, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.4724590480327606, "acc": 0.7698714733123779, "length": 3.9939494132995605, "mode": "train", "epoch": 14}
{"loss": 0.5218395590782166, "acc": 0.7521485686302185, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.46401792764663696, "acc": 0.7778658270835876, "length": 3.9948670864105225, "mode": "train", "epoch": 15}
{"loss": 0.5152297616004944, "acc": 0.7568359375, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.45469820499420166, "acc": 0.7843441367149353, "length": 3.994555950164795, "mode": "train", "epoch": 16}
{"loss": 0.5115822553634644, "acc": 0.7546874284744263, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.44609424471855164, "acc": 0.7911596298217773, "length": 3.996439218521118, "mode": "train", "epoch": 17}
{"loss": 0.515246570110321, "acc": 0.7578125, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.44200295209884644, "acc": 0.7914037704467773, "length": 3.9935429096221924, "mode": "train", "epoch": 18}
{"loss": 0.505652666091919, "acc": 0.7582030892372131, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.43581119179725647, "acc": 0.7931702136993408, "length": 3.9970703125, "mode": "train", "epoch": 19}
{"loss": 0.497333824634552, "acc": 0.7587890625, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.4307817220687866, "acc": 0.7991170287132263, "length": 3.9977469444274902, "mode": "train", "epoch": 20}
{"loss": 0.49534523487091064, "acc": 0.7652344107627869, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.4256288707256317, "acc": 0.800382137298584, "length": 3.99662709236145, "mode": "train", "epoch": 21}
{"loss": 0.48858702182769775, "acc": 0.7705078125, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.4206874668598175, "acc": 0.8011289238929749, "length": 3.9983253479003906, "mode": "train", "epoch": 22}
{"loss": 0.4951377511024475, "acc": 0.7662109732627869, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.414896696805954, "acc": 0.807556688785553, "length": 3.998133659362793, "mode": "train", "epoch": 23}
{"loss": 0.48628050088882446, "acc": 0.7701172828674316, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.4117393493652344, "acc": 0.8080940842628479, "length": 3.997947931289673, "mode": "train", "epoch": 24}
{"loss": 0.486065149307251, "acc": 0.7650390267372131, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.4064837694168091, "acc": 0.8125925660133362, "length": 3.998382329940796, "mode": "train", "epoch": 25}
{"loss": 0.4868350028991699, "acc": 0.7718749642372131, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.40713030099868774, "acc": 0.8146636486053467, "length": 3.9988479614257812, "mode": "train", "epoch": 26}
{"loss": 0.4961150884628296, "acc": 0.7775391340255737, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.40226322412490845, "acc": 0.8144959211349487, "length": 3.999638795852661, "mode": "train", "epoch": 27}
{"loss": 0.4866162836551666, "acc": 0.7791014909744263, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.3962806165218353, "acc": 0.8206827640533447, "length": 3.999601125717163, "mode": "train", "epoch": 28}
{"loss": 0.4788013696670532, "acc": 0.7873046398162842, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.38931477069854736, "acc": 0.8270317912101746, "length": 3.9980061054229736, "mode": "train", "epoch": 29}
{"loss": 0.4661126136779785, "acc": 0.7898436784744263, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.38504064083099365, "acc": 0.8310388922691345, "length": 3.9989230632781982, "mode": "train", "epoch": 30}
{"loss": 0.4696405827999115, "acc": 0.7941405773162842, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.38075563311576843, "acc": 0.8331180214881897, "length": 3.999333143234253, "mode": "train", "epoch": 31}
{"loss": 0.46054813265800476, "acc": 0.8011718392372131, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.3751358389854431, "acc": 0.837906539440155, "length": 3.9993159770965576, "mode": "train", "epoch": 32}
{"loss": 0.4508451223373413, "acc": 0.8078125715255737, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3677409887313843, "acc": 0.84046870470047, "length": 3.9992592334747314, "mode": "train", "epoch": 33}
{"loss": 0.44768285751342773, "acc": 0.8054686784744263, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.365510493516922, "acc": 0.8423495888710022, "length": 3.999826669692993, "mode": "train", "epoch": 34}
{"loss": 0.44819843769073486, "acc": 0.8033203482627869, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3633047640323639, "acc": 0.8439298272132874, "length": 3.9999072551727295, "mode": "train", "epoch": 35}
{"loss": 0.4421195983886719, "acc": 0.8082031011581421, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.35560622811317444, "acc": 0.8503866195678711, "length": 3.9989964962005615, "mode": "train", "epoch": 36}
{"loss": 0.44298166036605835, "acc": 0.8166016936302185, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.3558456599712372, "acc": 0.8499983549118042, "length": 3.9999501705169678, "mode": "train", "epoch": 37}
{"loss": 0.4532114267349243, "acc": 0.8101562857627869, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3494156002998352, "acc": 0.8531402349472046, "length": 3.9994654655456543, "mode": "train", "epoch": 38}
{"loss": 0.42881840467453003, "acc": 0.8220702409744263, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3440256714820862, "acc": 0.8552786707878113, "length": 3.9999849796295166, "mode": "train", "epoch": 39}
{"loss": 0.4426782429218292, "acc": 0.8082031607627869, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.3429238498210907, "acc": 0.8546109795570374, "length": 3.9999828338623047, "mode": "train", "epoch": 40}
{"loss": 0.4341224133968353, "acc": 0.8189453482627869, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.3391825556755066, "acc": 0.8562720417976379, "length": 3.9990015029907227, "mode": "train", "epoch": 41}
{"loss": 0.4350980818271637, "acc": 0.8232422471046448, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.3382587432861328, "acc": 0.8577609062194824, "length": 3.999863624572754, "mode": "train", "epoch": 42}
{"loss": 0.4308122396469116, "acc": 0.8130859732627869, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.3361359238624573, "acc": 0.8577666282653809, "length": 3.9999868869781494, "mode": "train", "epoch": 43}
{"loss": 0.43832722306251526, "acc": 0.8189453482627869, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.3327869176864624, "acc": 0.8612860441207886, "length": 3.9999868869781494, "mode": "train", "epoch": 44}
{"loss": 0.4306652545928955, "acc": 0.8167968392372131, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.3327285349369049, "acc": 0.8621964454650879, "length": 3.999997615814209, "mode": "train", "epoch": 45}
{"loss": 0.42875784635543823, "acc": 0.8187499642372131, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.32969072461128235, "acc": 0.8623178601264954, "length": 3.999995231628418, "mode": "train", "epoch": 46}
{"loss": 0.42120710015296936, "acc": 0.8238281607627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3306250274181366, "acc": 0.8614075183868408, "length": 3.9999988079071045, "mode": "train", "epoch": 47}
{"loss": 0.43154600262641907, "acc": 0.8087891936302185, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.33601564168930054, "acc": 0.8596301078796387, "length": 3.9996793270111084, "mode": "train", "epoch": 48}
{"loss": 0.42572516202926636, "acc": 0.8275390267372131, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.33980703353881836, "acc": 0.856178343296051, "length": 3.9997451305389404, "mode": "train", "epoch": 49}
{"loss": 0.42814597487449646, "acc": 0.8242188096046448, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.3279382884502411, "acc": 0.8613462448120117, "length": 3.999980926513672, "mode": "train", "epoch": 50}
{"loss": 0.4346289336681366, "acc": 0.8197265267372131, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.3284289240837097, "acc": 0.8599454164505005, "length": 3.9998416900634766, "mode": "train", "epoch": 51}
{"loss": 0.44165319204330444, "acc": 0.814453125, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.3258429765701294, "acc": 0.8634101152420044, "length": 3.999999523162842, "mode": "train", "epoch": 52}
{"loss": 0.4190819263458252, "acc": 0.8255859017372131, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.32197365164756775, "acc": 0.8640164136886597, "length": 3.999990463256836, "mode": "train", "epoch": 53}
{"loss": 0.4013294577598572, "acc": 0.8308592438697815, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.31786829233169556, "acc": 0.8655946850776672, "length": 3.999999761581421, "mode": "train", "epoch": 54}
{"loss": 0.41336584091186523, "acc": 0.8181639909744263, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.32047203183174133, "acc": 0.8649145364761353, "length": 3.999725580215454, "mode": "train", "epoch": 55}
{"loss": 0.41516730189323425, "acc": 0.8289061784744263, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.31262096762657166, "acc": 0.86874920129776, "length": 3.9999804496765137, "mode": "train", "epoch": 56}
{"loss": 0.41233572363853455, "acc": 0.8298827409744263, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.31918463110923767, "acc": 0.8676577210426331, "length": 3.9999988079071045, "mode": "train", "epoch": 57}
{"loss": 0.41776758432388306, "acc": 0.8222656846046448, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.31505200266838074, "acc": 0.8658978939056396, "length": 3.999997138977051, "mode": "train", "epoch": 58}
{"loss": 0.4126223921775818, "acc": 0.8316406607627869, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.3056534230709076, "acc": 0.8728761672973633, "length": 3.999999523162842, "mode": "train", "epoch": 59}
{"loss": 0.4130227565765381, "acc": 0.8267578482627869, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.3045186698436737, "acc": 0.8729367256164551, "length": 3.999999523162842, "mode": "train", "epoch": 60}
{"loss": 0.4115338921546936, "acc": 0.82421875, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.6867647767066956
